CONTRIBUTING.md

```diff
@@ -35,3 +35,41 @@ Examples:
 ./run-tests.sh 5.4 ./link
 ```
 
+# Contributor ladder
+
+If you'd like to contribute to the library more regularly, one of the
+[maintainers][ebpf-lib-maintainers] can add you to the appropriate team or mark
+you as a code owner.
+Just create an issue in the repository.
+
+## [ebpf-lib-contributors]
+
+Contributors support the development of the library by writing code, diagnosing
+bugs, triaging issues, answering questions, etc.
+
+* Have ["Triage"][roles] role
+* May be asked to review certain parts of code
+* May be asked to help with certain issues
+
+## [ebpf-lib-reviewers]
+
+Code owners have contributed a certain feature or are domain experts. They support
+development by helping to review code.
+
+* Have ["Write"][roles] role
+* Can't merge to protected branches
+* Can't create releases
+* May be asked to become CODEOWNERS
+
+## [ebpf-lib-maintainers]
+
+Code owners of last resort and responsible for the overall direction of the
+library.
+
+* Have ["Admin"][roles] role
+* Can re-run failed CI tasks
```

Comment:
Reviewers should be able to do this, but for that we probably need to migrate to GH actions

Reply:
That's fine for now, don't recall Semaphore getting stuck jobs lately.